packages/contracts-js/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "Apache-2.0",
   "repository": "https://github.com/thirdweb-dev/js/tree/main/packages/contracts-js",
   "dependencies": {
-    "@thirdweb-dev/contracts": "3.1.8-0"
```

Comment:
did u mean to update this in this branch?

Reply:
yep, its intentional - this was to get the new contracts that don't allow zero address